cmd/buck/cli/cli.go

```diff
@@ -97,3 +100,19 @@ func printLinks(reply *pb.LinksReply) {
 		cmd.Message("%s Bucket website", aurora.White(reply.WWW).Bold())
 	}
 }
+
+func setCidVersion(buck *local.Bucket, key string) {
+	if !buck.Path().Root().Defined() {
+		ctx, cancel := clients.Ctx.Thread(cmd.Timeout)
+		defer cancel()
+		root, err := clients.Buckets.Root(ctx, key)
+		if err != nil {
+			cmd.Fatal(err)
+		}
+		rp, err := util.NewResolvedPath(root.Root.Path)
+		if err != nil {
+			cmd.Fatal(err)
+		}
+		buck.SetCidVersion(int(rp.Root().Version()))
```

Comment:
OK, nice. Seems to answer my above question: The local bucket "local DAG" will use the same Cid version as the remote backend is using.